playground/frontend/playground_components/lib/src/services/symbols/loaders/map.dart

```diff
@@ -17,20 +17,33 @@
  */
 
 import 'package:highlight/languages/go.dart';
+import 'package:highlight/languages/java.dart';
 import 'package:highlight/languages/python.dart';
+import 'package:highlight/languages/scala.dart';
 
 import '../../../assets/assets.gen.dart';
 import '../../../playground_components.dart';
 import 'yaml.dart';
 
 final symbolLoadersByMode = {
+  //
   go: YamlSymbolsLoader(
     path: Assets.symbols.goG,
     package: PlaygroundComponents.packageName,
   ),
 
+  java: YamlSymbolsLoader(
+    path: Assets.symbols.javaG,
+    package: PlaygroundComponents.packageName,
+  ),
+
```

Comment:
Lets reuse this object so we do not read the 12k lines twice.

Reply:
done